<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLCDNumber" name="Ver_lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Omni_lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="Post_Label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Post</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Omni_lcdNumber">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Ver_lcdNumber">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="Ver_Label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Ver</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Post_lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="Omni_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Omni</string>
   </property>
  </widget>
  <widget class="QLabel" name="Post_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Post</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Hor_lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="Omni_Label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Omni</string>
   </property>
  </widget>
  <widget class="QLabel" name="Ver_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Ver</string>
   </property>
  </widget>
  <widget class="QLabel" name="DueNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Arm #1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Hor_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Hor</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Post_lcdNumber">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Hor_lcdNumber">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="DueNumber2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Arm # 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="Hor_Label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Hor</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
